types/hour.pp

```diff
@@ -2,8 +2,8 @@
 type Cron::Hour = Variant[
   Integer[0,23],
   Pattern[/(?x)\A(
-    \* ( \/ ( 1?[0-9]|2[0-3]|[3-9] ) )?
-    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[3-9] ) )? )?
-        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[3-9] ) )? )? )*
+    \* ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )?
+    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )? )?
+        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-9]|2[0-3]|[1-9] ) )? )? )*
```

Comment:
```suggestion
    \* ( \/ ( 1[0-2]|[1-9] ) )?
    |       ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-2]|[1-9] ) )? )?
        ( , ( 1?[0-9]|2[0-3] ) ( - ( 1?[0-9]|2[0-3] ) ( \/ ( 1[0-2]|[1-9] ) )? )? )*
```

Reply:
Doesn't 1-19/15 mean 1 and 16, which are valid hours? :wink:

Reply:
I expected `1-19/15` to mean `15` but you are right.

```
Following a range with ``/<number>'' specifies skips of the number's value through the range.
```

Forget this :upside_down_face: